common/src/main/java/com/wynntils/gui/render/MapRenderer.java

```diff
@@ -13,27 +13,44 @@
 import com.mojang.blaze3d.vertex.VertexFormat;
 import com.mojang.math.Matrix4f;
 import com.wynntils.features.user.overlays.map.PointerType;
+import com.wynntils.gui.screens.maps.MainMapScreen;
+import com.wynntils.mc.objects.CommonColors;
 import com.wynntils.mc.objects.CustomColor;
 import com.wynntils.mc.utils.McUtils;
+import com.wynntils.utils.Pair;
+import com.wynntils.wynn.model.map.MapModel;
 import com.wynntils.wynn.model.map.MapProfile;
+import com.wynntils.wynn.model.map.poi.LabelPoi;
+import com.wynntils.wynn.model.map.poi.Poi;
+import com.wynntils.wynn.model.map.poi.ServicePoi;
+import java.util.List;
 import net.minecraft.client.renderer.GameRenderer;
 import org.lwjgl.opengl.GL11;
 import org.lwjgl.opengl.GL13;
 
 public class MapRenderer {
+    private static Poi hovered = null;
+
     // TODO: Support circle map rendering
     public static void renderMapQuad(
             MapProfile map,
             PoseStack poseStack,
+            float mapCenterX,
+            float mapCenterZ,
```

Comment:
These 2 variables can be removed in all the remaining methods by translating before rendering

Reply:
I don't think so? They are used a lot in the code. Even if we could remove them, it might be clearer to leave them.

Reply:
Oops, I meant to refer to `centerX` and centerZ`